update whats-new.md with new pricing (#103604)
Azure/azure-content: articles/cognitive-services/Anomaly-Detector/whats-new.md

## Release notes

### Jan 2023

> * Multivariate Anomaly Detection starts charging since January 10th, 2023. The pricing details see in [pricing page](https://azure.microsoft.com/pricing/details/cognitive-services/anomaly-detector/).

Reviewer: ```suggestion
* Multivariate Anomaly Detection will begin charging as of January 10th, 2023. For pricing details see the [pricing page](https://azure.microsoft.com/pricing/details/cognitive-services/anomaly-detector/).
```
